enterprise/server/backends/pebble_cache/pebble_cache.go

```diff
@@ -176,6 +176,8 @@ type Options struct {
 	IncludeMetadataSize bool
 
 	Clock clockwork.Clock
+
+	ClearCacheBeforeMigration bool
```

Comment:
You can omit 'BeforeMigration' from the option name, it doesn't need to be tied to migrations.

Reply:
i changed to clear_cache_on start_up